README.md

```diff
@@ -77,7 +77,7 @@ You'll also need `npm` to download all the dependencies of the project.
 
 The general workflow / steps are below:
 
-```
+```bash
```

Comment:
Sorry, this is a drive-by change: I thought the setup instructions could be more readable when syntax-highlighted as Bash.

Reply:
nah this is a good change!